src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -1943,9 +1945,15 @@ namespace OutputReportTabular {
 					displayEconomicResultSummary = true;
 					WriteTabularFiles = true;
 					nameFound = true;
-				} else if ( SameString( AlphArray( iReport ), "EnergyMeters" ) ) {
+				} else if (SameString(AlphArray(iReport), "EnergyMeters")) {
 					WriteTabularFiles = true;
 					nameFound = true;
+				} else if (SameString(AlphArray(iReport), "EIO")) {
+					WriteTabularFiles = true;
```

Comment:
Should this be `displayEioSummary` since it is checking for "EIO"? I could be wrong.